src/main/java/Duke.java

```diff
@@ -1,10 +1,191 @@
+    }
+
+    public static void addNewEvent(Task[] tasks, String[] splitInput, int taskCount){
+        String at;
+        int indexOfAt = 0;
+        String description = getDescription(splitInput, indexOfAt);
+
+        for (int i = 0; i < splitInput.length; i++) {
+            if (splitInput[i].equals("/at")) {
+                indexOfAt = i;
+                break;
+            }
+        }
+        description = getDescription(splitInput, indexOfAt);
+        at = getDetails(splitInput, indexOfAt);
+        tasks[taskCount] = new Event(description, at);
+    }
+
+    public static String getDescription(String[] splitInput, int indexOfBy) {
+        String description = "";
+        for (int i = 0; i < indexOfBy; i++) {
+            description += splitInput[i] + " ";
+        }
+        return description;
+    }
+    public static String getDetails(String[] splitInput, int indexOfBy) {
+        String details = "";
+        for (int i = indexOfBy + 1; i < splitInput.length; i++) {
+            details += splitInput[i] + " ";
+        }
+        return details;
+    }
+
+    public static void printConfirmation(Task[] tasks, int taskCount) {
+        int numberOfTasks = taskCount + 1;
+        printSeparator();
+        System.out.println(" Got it, I've added this task: ");
+        System.out.println("   " + tasks[taskCount]);
+        System.out.println(" Now you have " + numberOfTasks + " task(s) in the list.");
+        printSeparator();
+    }
+
+
+    //Adds new task
+    public static void taskAdder(Task[] tasks, String description, int taskCount) {
+        Task newTask = new Task(description);
+        tasks[taskCount] = newTask;
+        printSeparator();
+        System.out.println(" added: " + description);
+        printSeparator();
+    }
+
+    //Sets isDone of Task object to be true
+    public static void markTaskAsDone(Task[] tasks, String description, int taskCount) {
+        //Determine index of task to be marked as done
+        String[] splitDescription = description.split(" ");
+        if(splitDescription.length == 1) {
+            System.out.println("Invalid command!");
+            printSeparator();
+            return;
+        }
+        int taskNumber = Integer.parseInt(splitDescription[1]) - 1;
+        printSeparator();
+        //Check if taskNumber is out of bounds
+        if(taskNumber < 0 || taskNumber > taskCount - 1) {
+            System.out.println("Invalid task number!");
+            printSeparator();
+            return;
+        }
+        //the taskNumber is valid
+        tasks[taskNumber].markAsDone();
+        System.out.println("Nice! I've marked this task as done: ");
+        System.out.println("  [" + "\u2713" + "] " + tasks[taskNumber].description);
+        printSeparator();
+    }
+
+    //Prints separator line
+    public static void printSeparator() {
+        System.out.println("______________________________________________________________________________");
```

Comment:
Comments like this can be removed since the method name and code is self explanatory.